examples/examples.sbt

```diff
@@ -2,11 +2,8 @@ name := "http4s-examples"
 
 description := "Common code for http4s examples on various backends"
 
-seq(Revolver.settings: _*)
-
 libraryDependencies ++= Seq(
-  logbackClassic,
-  jspApi % "runtime" // http://forums.yourkit.com/viewtopic.php?f=2&t=3733
+  "ch.qos.logback" % "logback-classic" % "1.1.7"
```

Comment:
No, but I didn't like an example project referring to something in the http4s build.  Then again, that's a weak justification, since this build.sbt still isn't standalone.

That JSP might be an issue still, too.  I just haven't used YourKit since I changed jobs.

Should I make it how it was?